freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/user/model/SynchronizeAllUsersRequest.java

```diff
@@ -24,6 +24,9 @@
     @ApiModelProperty(value = UserModelDescriptions.USERSYNC_ACCOUNT_ID)
     private String accountId;
 
+    @ApiModelProperty(value = UserModelDescriptions.FORCE_WORKLOAD_CREDENTIALS_UPDATE)
+    private Boolean forceWorkloadCredentialsUpdate = Boolean.FALSE;
```

Comment:
`Boolean` is not too flexible. Shouldn't we use some enum here so we could add other options if necessary?

Reply:
This seems like a simple on/off switch to me. I'm trying to think of other possible options, but not coming up with anything. Do you have some ideas of what the options might look like?

Reply:
force all, force only machine user, force only non machine users, force users with ssh-key. my point is that we saw similar cases that a boolean seems the right choice, as it can only be on/off and a few month later there came a third option. it gives us more options in the future as the API must be backward compatible.

Reply:
> force all, force only machine user, force only non machine users, force users with ssh-key. my point is that we saw similar cases that a boolean seems the right choice, as it can only be on/off and a few month later there came a third option. it gives us more options in the future as the API must be backward compatible.

That makes sense. I will change it to be an enum.

Reply:
Done.